Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -132,13 +138,23 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate {
     var url: NSURL? {
         didSet {
             lockImageView.hidden = (url?.scheme != "https")
-            if let url = url?.absoluteString {
-                if url.hasPrefix("http://") ?? false {
-                    locationLabel.text = url.substringFromIndex(advance(url.startIndex, 7))
-                } else if url.hasPrefix("https://") ?? false {
-                    locationLabel.text = url.substringFromIndex(advance(url.startIndex, 8))
+
+            if let urlString = url?.absoluteString {
+
+                var truncatedURL: String
```

Comment:
I think Swift now allows you to use `let` here (since it's guaranteed to be set once the `if` statement is done).